tests/e2e/commands/network.go

```diff
@@ -53,17 +54,42 @@ func StartNetwork() string {
 	return StartNetworkWithVersion("")
 }
 
+func StartNetworkWithNodeNumber(numOfNodes uint) string {
```

Comment:
We should probably start using Raymond's test function so that we don't create these additional utils functions for every scenario

Reply:
good reminder

Reply:
addressed